keyboards/drakon/keymaps/wkl/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2020 jagdpietr
```

Comment:
This file can be deleted.

Reply:
Done!

Reply:
The files are still there.

Reply:
I actually deleted that line already, but I wonder why it is still there. If I edit it too that line is not seen too. But I will do it again.
Thank you for looking through all of these!

Reply:
Not the line, the entire file needs to be deleted.

Reply:
Finished the deletion